frontend/src/styles/tokens.css

```
@@ -0,0 +1,42 @@
+  --fs-panel: 15px;
+  --fs-metric: 20px;
+  --fs-display: 26px;
+
+  --sp-1: 4px;
+  --sp-2: 8px;
+  --sp-3: 12px;
+  --sp-4: 16px;
+  --sp-5: 24px;
+  --sp-6: 32px;
+
+  --font-mono: ui-monospace, SFMono-Regular, Consolas, "Liberation Mono", monospace;
```

Stylelint (17.14.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)